src/PrestaShopBundle/Form/Admin/Sell/Order/AddProductRowType.php

```diff
@@ -96,14 +96,14 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     'class' => 'custom-select',
                 ],
             ])
-            ->add('price_tax_excluded', TextWithUnitType::class, [
+            ->add('price_tax_excluded', NumberWithUnitType::class, [
```

Comment:
Shouldn't this be a `MoneyType` here?

Reply:
Thank you @jolelievre i add your suggestion, for MoneyType we have to check if it's work good.

It's a Cooperative Pull Request

Reply:
We need to use NumberWithUnitType for TTC and HT informations

Reply:
Why can't you use `MoneyType` here? It's supposed to be the best fit for prices because it also includes some internal internationalization feature to display the appropriate symbol and stuff like that

Reply:
@jolelievre so we need to create `MoneyWithUnitType` here?

Reply:
It seems `MoneyWithSuffixType` already exists